src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -110,86 +242,129 @@ dx_path() {
       ;;
   esac
 }
-    
+
 main() {
-    if [[ $debug == true ]]; then
-      export NXF_DEBUG=2
-      TRACE_CMD="-trace nextflow.plugin"
-      set -x && env | grep -v DX_SECURITY_CONTEXT | sort
-    fi
-    
-    if [ -n "$docker_creds" ]; then
-        dx-registry-login
-    fi
+  if [[ $debug == true ]]; then
+    export NXF_DEBUG=2
+    TRACE_CMD="-trace nextflow.plugin"
+    env | grep -v DX_SECURITY_CONTEXT | sort
+    set -x
+  fi
+
+  DX_CACHEDIR=$DX_PROJECT_CONTEXT_ID:/.nextflow_cache_db
+  validate_run_opts
+
+  if [[ $preserve_cache == true ]]; then
+    check_cache_db_storage
+  fi
+
+  if [ -n "$docker_creds" ]; then
+    dx-registry-login
+  fi
+
+  # set default NXF env constants
+  export NXF_DOCKER_LEGACY=true
+  #export NXF_DOCKER_CREDS_FILE=$docker_creds_file
+  #[[ $scm_file ]] && export NXF_SCM_FILE=$(dx_path $scm_file 'Nextflow CSM file')
+  export NXF_HOME=/opt/nextflow
+  export NXF_ANSI_LOG=false
+  export NXF_PLUGINS_DEFAULT=nextaur@1.1.0
+  export NXF_EXECUTOR='dnanexus'
+
+  # use /home/dnanexus/nextflow_playground as the temporary nextflow execution folder
+  mkdir -p /home/dnanexus/nextflow_playground
```

Comment:
How about we gave it a more "non-temporary-sounding" name, like `/home/dnanexus/nextflow_execution`?